main.py

```diff
@@ -112,6 +112,12 @@ def get(self):
 
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
+    Arg:
```

Comment:
Undo the changes in this file. (I don't think this is an expected change, right?)